README.md

```diff
@@ -1,13 +1,13 @@
-<p>
+<p align="center">
```

Comment:
Heh, I actually moved this to the left because I thought it looked better.  If centered is what people want, I'm ok with that :)